packages/stylelint-plugin/types/stylelint.d.ts

```diff
@@ -0,0 +1,187 @@
+// Type definitions for stylelint 9.10
+// Project: https://github.com/stylelint/stylelint, https://stylelint.io
+// Definitions by: Alan Agius <https://github.com/alan-agius4>
+//                 Filips Alpe <https://github.com/filipsalpe>
+//                 James Garbutt <https://github.com/43081j>
+//                 Bob Matcuk <https://github.com/bmatcuk>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
```

Comment:
Is there a reason you're not using the package from definitely-typed?

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/stylelint/index.d.ts

Reply:
(If there is a reason, it would be good to include a comment explaining why and any local changes you've made)